src/main/java/com/shapesecurity/salvation/directives/Directive.java

```diff
@@ -85,6 +85,8 @@ static void register(Class<? extends Directive> directiveClass) {
         set.addAll(a);
         set.addAll(b);
 
+        set = set.stream().filter(x -> x != None.INSTANCE).collect(Collectors.toCollection(LinkedHashSet::new));
```

Comment:
Just remove it. It's like `set.delete(None.INSTANCE)` or something.

Reply:
Done